src/game/timing.c

```diff
@@ -39,8 +39,8 @@ void frametimeCalculate(void)
 		diffframet = count - g_Vars.thisframestartt;
 		g_Vars.diffframet = diffframet;
 
-		diffframe60 = (g_Vars.lostframetime60t + diffframet + CYCLES_PER_FRAME / 2) / CYCLES_PER_FRAME;
-		diffframe240 = (g_Vars.lostframetime240t + diffframet + CYCLES_PER_FRAME / 2 / 4) / (CYCLES_PER_FRAME / 4);
+		diffframe60 = (diffframet + CYCLES_PER_FRAME / 2) / CYCLES_PER_FRAME;
+		diffframe240 = (diffframet + CYCLES_PER_FRAME / 2 / 4) / (CYCLES_PER_FRAME / 4);
 	} while (diffframe60 < g_Vars.mininc60);
 
 	g_Vars.lostframetime60t = g_Vars.lostframetime60t + diffframet - diffframe60 * CYCLES_PER_FRAME;
```

Comment:
This fixes the random jittering without the combat boost.
I don't fully understand what is the use for this `g_Vars.lostframetime60t` here, but i noticed that the way its calculated is actually redundant (I wonder if this is one of the decompilation quirks). 
For clarity, lets name`g_Vars.lostframetime60t as L`, `diffframet as D` and `diffframe60 as D60`. In line 46, if we expand diffframe60 to the value from line 42, we get:
`L = L + D - ((L + D + CYCLES_PER_FRAME/2) / CYCLES_PER_FRAME) * CYCLES_PER_FRAME`
This whole thing simplifies to `-CYCLES_PER_FRAME/2`. So in the next frame when we plug that into line 42, it will cancel out with the term `CYCLES_PER_FRAME/2`.